src/signature-help.jl

```diff
@@ -329,7 +327,7 @@ function call_is_decl(_bas::JL.SyntaxList, i::Int, _basᵢ::JL.SyntaxTree = _bas
     return j <= lastindex(_bas) &&
         kind(_bas[j]) in JS.KSet"macro function" &&
         # in `f(x) = g(x)`, return true in `f`, false in `g`
-        _bas[j - 1] === _bas[j][1]
+        _bas[j - 1]._id === _bas[j][1]._id
```

Comment:
nitpick: For comparing numeric characters, it's generally better to use `==` since it can handle cases where the numeric values have different types but represent the same value, e.g. `0x01 == 1`. Of course, in this case I believe the `_id` field is probably always `::Int`, so `===` would also work fine.